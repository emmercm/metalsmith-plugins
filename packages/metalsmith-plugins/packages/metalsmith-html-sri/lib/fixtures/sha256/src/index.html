<!DOCTYPE html>
<html lang="en">
<head>
  <link rel="stylesheet" href="style.css">
  <link rel="stylesheet" href="style.css">
  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.8.2/css/all.css">
  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
</head>
<body>
<script src="script.js"></script>
<script src="script.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"></script>
</body>
</html>
