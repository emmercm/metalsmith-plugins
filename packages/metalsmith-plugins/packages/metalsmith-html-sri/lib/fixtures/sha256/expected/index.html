<!DOCTYPE html><html lang="en"><head>
  <link rel="stylesheet" href="style.css" integrity="sha256-oG/XUN5zdJg9r0ABZWSx+28haO0sV0LM9pkS6FdIA8A=">
  <link rel="stylesheet" href="style.css" integrity="sha256-oG/XUN5zdJg9r0ABZWSx+28haO0sV0LM9pkS6FdIA8A=">
  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-YLGeXaapI0/5IgZopewRJcFXomhRMlYYjugPLSyNjTY=" crossorigin="anonymous">
  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.8.2/css/all.css">
  <link href="https://fonts.googleapis.com/css?family=Roboto&amp;display=swap" rel="stylesheet">
</head>
<body>
<script src="script.js" integrity="sha256-oouMTC+GBbmDnmCcd+j9YjXWnGePm/01/VUI5xCcvmI="></script>
<script src="script.js" integrity="sha256-oouMTC+GBbmDnmCcd+j9YjXWnGePm/01/VUI5xCcvmI="></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js" integrity="sha256-fzFFyH01cBVPYzl16KT40wqjhgPtq6FFUB6ckN2+GGw=" crossorigin="anonymous"></script>


</body></html>